.github/CODEOWNERS

```diff
@@ -195,4 +195,3 @@ packages/react-experiments/src/components/Pagination/ @caradong
 packages/react-experiments/src/components/Signals/ @ThomasMichon
 packages/react-experiments/src/components/Tile/ @ThomasMichon
 packages/react-experiments/src/components/TileList/ @ThomasMichon
-packages/react-experiments/src/components/Toggle/ @khmakoto
```

Comment:
should this be @microsoft/cxe-red too? I guess it's not super important because it's in experiments

Reply:
I removed it since it does not exist anymore in v8 😄

Reply:
For the ones above I think it does not matter as much since they are in experiments like you say.

Reply:
oh yeah... the color of the line was red... 😆 got it, yeah I agree on experiments can be owned by anyone

Reply:
P.S. sorry for the delay on approval, was going to approve anyway inspite of my comment, but then *squirrel* and I didn't